### DISCUSSION TITLE
UI Feedback

### DISCUSSION THREAD

**Discussion by pnbrown** [vmware-tanzu/community-edition#2632] [General]
Hi All,
This tread is to follow up on the discussion from our [December 1st Community Meeting](https://youtu.be/fZN7V8T6GIc) about the UI in Tanzu Community Edition. Please leave your thoughts here.

cc: @miclettej @bhattm-vmware

**Comment by miclettej** [comment]
Some of the ideas raised during the discussion:
- UI should allow user to create workload clusters
- UI should not be embedded in management-cluster CLI plugin. For the UI to mature into a more holistic solution, it should be able to be run outside of a CLI plugin that is specific to a vertical slice of functionality (e.g. management cluster creation)
- Provide first time users with a more guided experience, while allowing opt-out for experienced users. This could include helping users better understand cloud provider requirements and configuration (IE AWS credentials), understanding pre-requisites, providing more pre-requisite checks/validation.
- Integration of contextual help topics (user can click a "more info" link or icon near a field or set of fields to reveal in-app contextual help)
